src/main/resources/application.yml

```diff
@@ -79,3 +83,5 @@ cwa-testresult-server:
     key-store-password: changeit
     trust-store: classpath:truststore.jks
     trust-store-password: changeit
+server:
```

Comment:
really needed?

Reply:
I have removed these 2 lines. I've added them for testing purpose and have forgotten to remove again